cypher-shell/src/main/java/org/neo4j/shell/Main.java

```diff
@@ -27,14 +25,32 @@ public static void main(String[] args) {
         main.startShell(cliArgs);
     }
 
+    private static void printWelcomeMessage(@Nonnull Logger logger,
```

Comment:
I just hit auto format on all my changed files and IntelliJ thought this should be listed first, most likely because it's a static method